docs/build-walkthrough/build-process.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Development Guide
+sidebar_label: Development Guide
+---
+
+To deploy a synthetic token and start receiving developer mining rewards, it recommended that you follow the docs tutorials in this order.
```

Comment:
minor grammar - "it recommended" to "it is recommended"

Reply:
Good catch, thanks! Addressed